ui/scss/component/_markdown-preview.scss

```diff
@@ -128,6 +128,10 @@
     }
   }
 
+  .img__zoomable {
+    cursor: pointer; // 'zoom-in' would be ideal, but browser-dependant.
```

Comment:
This could be solved down the road by adding a hover overlay to the image that indicates zoom-in.

https://stackoverflow.com/questions/21086385/how-to-make-in-css-an-overlay-over-an-image

https://stackoverflow.com/questions/21086385/how-to-make-in-css-an-overlay-over-an-image